src/config/mocks/adapters/diners.js

```diff
@@ -0,0 +1,32 @@
+import diners from '../data/diners';
+import { extractIdPathParamFromUrl, createPaginationResponse } from '../common/helpers';
+
+const getDiners = (config) => {
+  console.log('getDiners mock');
```

Comment:
Destroy

Reply:
This console logs are intended since will notify on the console that the mock is being used instead of the actual call